lib/main_common.pm

```diff
@@ -892,6 +892,10 @@ sub load_inst_tests {
             if (get_var("EXPERTPARTITIONER")) {
                 loadtest "installation/partitioning_expert";
             }
+            #kernel performance test need install on specific disk
+            if (get_var("SPECIFICDISK")) {
+                loadtest "installation/partitioning_olddisk";
```

Comment:
The file name is a little confusing, maybe rename it to a better one which  can reflect clearer purpose?

Reply:
Updated as "SPECIFIC_DISK"